atlp-rwanda/e-commerce-lydia-32-fn
# Ft register and verify email 187570815

### What does this PR do?
This PR implements the signup and email verification feature.
### Description of Task to be completed?

1. Implement user signup functionality
2. Implement email verification process

> Send verification email upon successful signup
> Create endpoint for email verification
> Update user status to verified upon successful email verification

### How should this be manually tested?
1. Clone the repository and navigate to the project directory.
3. Install dependencies using npm install.
4. Start the app using npm start or yarn start.
5. Open URL in any Browser to test.
6. Test the signup filling required fields 
7. Check the email inbox of the provided email address for the verification email.
8. Click the link ion the email to be verified

#### N/A

### Any background context you want to provide?

- The verification email includes a link with a token that is used to verify the user's email address.
- The token is valid for a limited time 

#### N/A

### Screenshots (if appropriate)

#### N/A

### Questions:

#### N/A

### What are the relevant pivotal tracker stories

#### N/A
